ledger/src/blockstore.rs

```diff
@@ -3476,6 +3492,30 @@ fn handle_chaining(
     Ok(())
 }
 
+/// A helper function of handle_chaining which handles the chaining based
+/// on the `SlotMetaWorkingSetEntry` of the specified `slot`.  Specifically,
+/// it handles the following two things:
+///
+/// First, based on the `SlotMetaWorkingSetEntry` for `slot`, check if `slot`
+/// did not previously have a parent slot but does now.  If `slot` satisfies
+/// this condition, update the Orphan property of both `slot` and its parent
+/// slot based on their current orphan status.
```

Comment:
"based on their current orphan status"

`slot` cannot be an orphan if it has a parent. So it might be clearer to say:

 1) "updates the orphan property of `slot` to no longer be an orphan because it has a parent"
2) adds to parent to the orphan column family if the parent's parent is currently unknown